include/urdf_model/joint.h

```diff
@@ -70,13 +71,19 @@ class JointLimits
   double upper;
   double effort;
   double velocity;
+  double acceleration;
+  double deceleration;
+  double jerk;
```

Comment:
keep in mind that the addition of these data members will change the size of this object and hence break ABI. This change may be merged to rolling if appropriate care is taken, but it should not be backported to any released ROS distros